autoload/go/path.vim

```diff
@@ -139,9 +139,15 @@ function! go#path#CheckBinPath(binpath)
     " remove whitespaces if user applied something like 'goimports   '
     let binpath = substitute(a:binpath, '^\s*\(.\{-}\)\s*$', '\1', '')
 
+    " append our GOBIN and GOPATH paths and be sure they can be found there...
+    " let us search in our GOBIN and GOPATH paths
+    let old_path = $PATH
+    let $PATH = go_bin_path . go#util#PathListSep() . $PATH
```

Comment:
Yes, it is, sorry. I just realized I had my rtp set to something else when I was doing this. pushing a fish now.